.github/workflows/flaky-test-monitor.yml

```diff
@@ -1,14 +1,20 @@
-name: Flaky Test Summary
+name: Flaky Test Monitor
 
 on:
+  schedule:
+    - cron: '0 */4 * * *' # every 4 hours
   push:
     branches:
       - master
+  pull_request: # TODO: remove this before merging
+    branches:
+      - master
 
 env:
   NUM_RUNS: 1
   GCS_BUCKET: production_ddp_flow_test_metrics
   COMMIT_SHA: ${{ github.sha }}
+  BIGQUERY_TABLE: dev_src_flow_test_metrics.flaky_test_monitor_level_1 # TODO: update to production_src_flow_test_metrics before merging
```

Comment:
why would we change the table before merging? I thought we would publish to the dev table for now